examples/conversion_amber_gromacs/1_cyclohexane_1_ethanol.pdb

```diff
@@ -0,0 +1,61 @@
+REMARK   1 CREATED WITH MDTraj 1.7.2, 2016-06-27
```

Comment:
As a note to other readers: I was initially on the fence about adding copies of the molecule files to the example folders, but I think users will find it much clearer to have the molecules in here where they can investigate them.